### PR TITLE
Added explanations to contribute.

### PR DESCRIPTION
**Pull request by ouphi**
Added explanations to contribute.
PR checks are not passing because setting up CI/CD is out of scope and is part of [this story](https://github.com/agera-edc/MinimumViableDataspace/issues/160)

Discussions link do not work yet, it will be enabled by Jim and Paul later.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
+Contributing to the Project
+===================================
+
+Thank you for your interest in contributing to the IdentityHub!
```
Comment (Izzzu) [inline CONTRIBUTING.md]: IdentityHub here is intended? It's MVD repository

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+before raising an issue.
+
+Please note that if an issue covers a topic or the response to a question that may be interesting 
+for other developers or contributors, or for further discussions, it should be converted to a 
+discussion and not be closed.
+
+### Adhere to Coding Style Guide
+
+We aim for a coherent and consistent code base, thus the coding style detailed in the 
+[styleguide](styleguide.md) should be followed.
```
Comment (cpeeyush) [inline CONTRIBUTING.md:76]: broken link ?

File: CONTRIBUTING.md (hunk 1 above)
Comment (ouphi) [reply CONTRIBUTING.md]: No it is not intended, I fixed it. Thanks!

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+
+### Submit a Pull Request
+
+In addition to the contribution guideline made available in the 
+[Eclipse project handbook](https://www.eclipse.org/projects/handbook/#contributing),
+we would appreciate if your pull request applies to the following points:
+
+* Conform to [Pull-Request Etiquette](pr_etiquette.md)
```
Comment (cpeeyush) [inline CONTRIBUTING.md:84]: broken link.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+  include a ["Signed-off-by" entry](https://wiki.eclipse.org/Development_Resources/Contributing_via_Git).
+  
+* Add meaningful tests to verify your submission acts as expected.
+
+* Where code is not self-explanatory, add documentation providing extra clarification.
+
+* Add documentation files to new modules. See [here](#add-documentation) for more details.
+
+* Add relevant changes (e.g., no typo fixes, updated readme files, fixes of stuck test) to the 
+  [changelog](CHANGELOG.md). If these refer to a new feature, add this to the `Overview` section
```
Comment (cpeeyush) [inline CONTRIBUTING.md:125]: broken link.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+
+* Add relevant changes (e.g., no typo fixes, updated readme files, fixes of stuck test) to the 
+  [changelog](CHANGELOG.md). If these refer to a new feature, add this to the `Overview` section 
+  and add your changes to the `Detailed Changes` section according to the rules documented on 
+  <http://keepachangelog.com>. Include more information via linking to existing pull requests,
+  issues, or discussions.
+
+* If a new module has been added or a significant part of the code has been changed and you should 
+  or want to be seen as the contact person for any further changes, please add appropriate 
+  information to the [CODEOWNERS](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/blob/main/CODEOWNERS)
```
Comment (cpeeyush) [inline CONTRIBUTING.md]: this link points to EDC, should be like that or needs to point to MVD instead ?

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+  issues, or discussions.
+
+* If a new module has been added or a significant part of the code has been changed and you should 
+  or want to be seen as the contact person for any further changes, please add appropriate 
+  information to the [CODEOWNERS](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/blob/main/CODEOWNERS) 
+  file. You can find instructions on how to do this at <https://help.github.com/articles/about-codeowners/>.
+  Please note that this file does not represent all contributions to the code. What persons and organizations
+  actually contributed to each file can be seen on GitHub and is documented in the license headers.
+
+* PR descriptions should use the current [PR template](.github/PULL_REQUEST_TEMPLATE.md)
```
Comment (cpeeyush) [inline CONTRIBUTING.md:137]: broken link.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,229 @@
...
+be qualified as abuse). If an issue receives no attention, there usually
+are reasons for it. It is therefore advisable to clarify in advance whether any particular feature fits into EDC's
+planning schedule and roadmap. For that, we recommend opening a discussion. Discussions serve us as a system of record, that 
+means we monitor them more closely, and do not close them automatically.
+
+### Add Documentation
+
+Every decision record, launcher, extension, or any type of module has to provide documentation that covers at least
+one markdown file with necessary information. Please find appropriate templates that should
+be used in [the templates directory](docs/templates).
```
Comment (cpeeyush) [inline CONTRIBUTING.md:168]: broken link.

**Review by cpeeyush — APPROVED** [review]
Approved with suggestions.

**Review by Izzzu — APPROVED** [review]

File: CONTRIBUTING.md (hunk 5 above)
Comment (ouphi) [reply CONTRIBUTING.md]: Thanks Peeyush! For now I changed it to point to the local file.

**Pull request merged**